manifests/claudie/ansibler.yaml

```diff
@@ -3,16 +3,22 @@ kind: Deployment
 metadata:
   name: ansibler
   labels:
+    app.kubernetes.io/part-of: claudie
     app: ansibler
```

Comment:
Please remove the old labels if they are "replaced" by the new ones.
This one seems to be replaced by `app.kubernetes.io/name: ansibler`

Same for all other occurrences, please